lib/types.js

```diff
@@ -51,7 +51,7 @@ declare class BabelNodeAssignmentExpression extends BabelNode {
 
 declare class BabelNodeBinaryExpression extends BabelNode {
   type: "BinaryExpression";
-  operator: string;
+  operator: "+" | "-" | "/" | "%" | "*" | "**" | "&" | "|" | ">>" | ">>>" | "<<" | "^" | "==" | "===" | "!=" | "!==" | "in" | "instanceof" | ">" | "<" | ">=" | "<=";
```

Comment:
I ran `scripts/generate-interfaces.js` and it changed all this stuff, I guess that script was updated but not ran recently. :stuck_out_tongue: